tests/unit/test_report_junit.py

```diff
@@ -39,10 +42,74 @@ def get(key, default=None):
     return report, results, out_file_path
 
 
-def do_assert(out_file_path, expected):
-    with open(out_file_path) as f:
-        output = f.read().replace('\t', ' ' * 4)
-    assert output == expected
+def _compare_xml_node(tree_path: List[str], expected: xml.dom.Node, actual: xml.dom.Node) -> None:
```

Comment:
@happz Can we add a note about python3.6 here, so future readers of the test understand why we had to change the way we compare/assert the correct output?

Reply:
Done.